Source/Core/DolphinQt2/ToolBar.cpp

```diff
@@ -56,10 +56,20 @@ void ToolBar::OnEmulationStateChanged(Core::State state)
   m_screenshot_action->setEnabled(running);
 
   bool playing = running && state != Core::State::Paused;
-  m_play_action->setEnabled(!playing);
-  m_play_action->setVisible(!playing);
-  m_pause_action->setEnabled(playing);
-  m_pause_action->setVisible(playing);
+  if (!playing)
```

Comment:
The true case should be first instead of the negated one. This can also be put in its own function (UpdatePausePlayButtonState())